src/utils/capitalize.ts

```diff
@@ -0,0 +1,4 @@
+const capitalize = (value: string) =>
```

Comment:
Could be done with css ?

Reply:
Not linked to style, I need capitalized variable for accessing to object such as `backgroudWeak`

Reply:
Not linked to style, I need capitalized variable for accessing to object such as `backgroudWeak`